doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -714,7 +714,11 @@ Bluetooth libraries and services
 Common Application Framework
 ----------------------------
 
-|no_changes_yet_note|
+* :ref:`caf_buttons`:
+
+  * Added possibility of using more GPIOs.
+    Earlier only ``GPIO0`` and ``GPIO1`` were supported.
```

Comment:
```suggestion
    Earlier, only **GPIO0** and **GPIO1** pins were supported.
```

Reply:
I'm changing it to **GPIO1** devices as this is more accurate. GPIOx device can control multiple pins that belongs to it.

Reply:
> I'm changing it to `**GPIO1** devices` as this is more accurate. GPIOx device can control multiple pins that belongs to it.